presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/MetadataDao.java

```diff
@@ -131,6 +132,16 @@ long insertTable(
             @Bind("compactionEnabled") boolean compactionEnabled,
             @Bind("distributionId") Long distributionId);
 
+    @SqlUpdate("INSERT INTO tables (schema_name, table_name, compaction_enabled, organization_enabled, distribution_id)\n" +
+            "VALUES (:schemaName, :tableName, :compactionEnabled, :organizationEnabled, :distributionId)")
+    @GetGeneratedKeys
+    long insertTable(
```

Comment:
Don't add a new method. Instead, add the parameter to the existing one.

Alternatively, if this is really only for testing, it should go in the test code, but I don't see a reason to do that here.